src/components/MainWindow/index.tsx

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
-import {useRecoilValue} from 'recoil';
-import { ErrorBoundary } from 'react-error-boundary';
+import {useAtomValue} from 'jotai';
 
-import ErrorFallbackComponent from '@src/components/common/ErrorFallback';
-import selectedSuperRegionAtom from '@src/state/client/derived/selectedSuperRegion';
+import {selectedSuperRegionAtom} from '@src/state/client/derived/selectedSuperRegion';
 import TileLayout from './TileLayout';
 import SuperRegionSplashSelector from './SuperRegionSplashSelector';
 
 const MainWindow: React.FC = () => {
-  const selectedSuperRegion = useRecoilValue(selectedSuperRegionAtom);
+  const selectedSuperRegion = useAtomValue(selectedSuperRegionAtom);
 
   const MainComponent = selectedSuperRegion ? TileLayout : SuperRegionSplashSelector;
   return (
-    <ErrorBoundary FallbackComponent={ErrorFallbackComponent}>
```

Comment:
- [ ] Re-add the main window error boundary, this is useful for errors that could be recovered with the control panel.